uiowa/uiowa
# Move SiteNow "Additional Info" content into uids_base

Resolves #1820 

## Testing
- Run update hook on a uiowa_bootstrap site.
- (maybe) Add some info to the Additional Info fields in basic site settings
- Switch theme to uids_base.
- If site used Additional Info
  - Content (title and/or details) should now be in a block in the footer, below the Contact Info
- If site wasn't using Additional Info
  - No change to the footer

Two SiteNow sites currently using the Additional Info fields are protostudios.uiowa.edu and igec.prod.drupal.uiowa.edu.

Note: There is some basic regexp logic to add link/icon to phone numbers which are not currently linked, to match default content styling/usage, and to remove lone instances of the site title to avoid repetition in the uids_base footer. It's not robust, and only a "nice if it works" addition, so not needed if it breaks.
